go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -178,10 +206,17 @@ func (se *Engine) EnsureConnectionAndDB(tabletType topodatapb.TabletType) error
 	ctx := tabletenv.LocalContext()
 	conn, err := dbconnpool.NewDBConnection(ctx, se.env.Config().DB.AllPrivsWithDB())
 	if err == nil {
+		log.Infof("inside EnsureConnectionAndDB with TabletType %v...", tabletType)
 		se.dbCreationFailed = false
 		// upgrade _vt if required, for a tablet with an existing database
 		if tabletType == topodatapb.TabletType_PRIMARY {
-			if err := se.syncVTDatabase(ctx, conn); err != nil {
+			dbaConn, err := dbconnpool.NewDBConnection(ctx, se.env.Config().DB.DbaConnector())
+			if err != nil {
+				log.Info("Not able to get connection with all privileges...")
```

Comment:
There's actually an AllPrivs user, but we're using the DBA user here. So this message is misleading.